rxandroidble/src/main/java/com/polidea/rxandroidble2/ConnectionSetup.java

```diff
@@ -32,19 +33,26 @@
      * canceled and removed from queue.
      */
     public final Timeout operationTimeout;
+    /**
+     * Timeout in seconds after which the connection will be considered as broken. Eventually the connection will be
+     * canceled and removed from queue.
+     */
+    public final Timeout connectTimeout;
 
-    private ConnectionSetup(boolean autoConnect, boolean suppressOperationCheck, Timeout operationTimeout) {
+    private ConnectionSetup(boolean autoConnect, boolean suppressOperationCheck, Timeout operationTimeout,
+                            Timeout connectTimeout) {
         this.autoConnect = autoConnect;
         this.suppressOperationCheck = suppressOperationCheck;
         this.operationTimeout = operationTimeout;
+        this.connectTimeout = connectTimeout;
```

Comment:
It's just a detail, but I wonder if instead of `connectTimeout` and `DEFAULT_CONNECT_TIMEOUT` shouldn't be `connectionTimeout` and `DEFAULT_CONNECTION_TIMEOUT` like there already are `DEFAULT_OPERATION_TIMEOUT` and `operationTimeout`